function-support.html

```diff
@@ -1234,10 +1234,10 @@ <h2 id="macros">Macros</h2>
 			<td>\(\gdef\bar#1{#1^2}&nbsp;\bar{y} + \bar{y}\)</td>
 			<td><strong>\gdef</strong>\bar#1{#1^2}&nbsp;\bar{y} + \bar{y}</td>
 		</tr>
-		<tr> <td></td>  <td>\global\def\macroname[args]{definition}</td> </tr>
-		<tr> <td></td>  <td>\newcommand\macroname[args]{definition}</td>  </tr>
-		<tr> <td></td>  <td>\renewcommand\macroname[args]{definition}</td>  </tr>
-		<tr> <td></td>  <td>\providecommand\macroname[args]{definition}</td>  </tr>

```

Comment:
This is wrong: `\newcommand`, `\renewcommand`, `\providecommand` use `[numargs]` not `#1#2...`.